connectors/microsoft-teams/src/main/java/io/camunda/connector/model/request/data/CreateChannel.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. Licensed under a proprietary license.
+ * See the License.txt file for more information. You may not use this file
+ * except in compliance with the proprietary license.
+ */
+package io.camunda.connector.model.request.data;
+
+import io.camunda.connector.generator.java.annotation.TemplateProperty;
+import io.camunda.connector.generator.java.annotation.TemplateSubType;
+import io.camunda.connector.model.MSTeamsMethodTypes;
+import jakarta.validation.constraints.NotBlank;
+
+@TemplateSubType(label = "Create channel", id = MSTeamsMethodTypes.CREATE_CHANNEL)
+public record CreateChannel(
+    @NotBlank
+        @TemplateProperty(
+            group = "data",
+            id = "createChannel.groupId",
+            label = "Group ID",
+            description = "The group ID for teams")
+        String groupId,
+    @NotBlank
+        @TemplateProperty(
+            group = "data",
+            id = "createChannel.name",
+            label = "Display name",
+            description = "Enter name of a channel")
+        String name,
+    @TemplateProperty(
+            group = "data",
+            id = "createChannel.description",
+            label = "Description",
+            optional = true,
+            type = TemplateProperty.PropertyType.Text,
+            description = "Enter description")
+        String description,
+    @NotBlank
+        @TemplateProperty(
+            group = "data",
+            id = "createChannel.channelType",
+            label = "Channel membership type",
+            type = TemplateProperty.PropertyType.Dropdown,
+            defaultValue = "standard",
+            choices = {
+              @TemplateProperty.DropdownPropertyChoice(value = "standard", label = "STANDARD"),
+              @TemplateProperty.DropdownPropertyChoice(value = "private", label = "PRIVATE"),
+              @TemplateProperty.DropdownPropertyChoice(value = "shared", label = "SHARED")
```

Comment:
Let's have dropdown labels 'normalized':

```suggestion
              @TemplateProperty.DropdownPropertyChoice(value = "standard", label = "Standard"),
              @TemplateProperty.DropdownPropertyChoice(value = "private", label = "Private"),
              @TemplateProperty.DropdownPropertyChoice(value = "shared", label = "Shared")
```

Here and everywhere else where applicable.